tests/client.py

```diff
@@ -403,6 +425,501 @@ def get_workflows(self):
 
         return records
 
+    ##########################################################################
+    ### CREATE
+    ##########################################################################
+
+    def create_contacts(self):
+        """
+        Generate a single contacts record.
+        Hubspot API https://legacydocs.hubspot.com/docs/methods/contacts/create_contact
+        """
+        record_uuid = str(uuid.uuid4()).replace('-', '')
+
+        url = f"{BASE_URL}/contacts/v1/contact"
+        data = {
+            "properties": [
+                {
+                   "property": "email",
+                   "value": f"{record_uuid}@stitchdata.com"
+                 },
+                 {
+                   "property": "firstname",
+                   "value": "Yusaku"
+                 },
+                 {
+                   "property": "lastname",
+                   "value": "Kasahara"
+                 },
+                 {
+                   "property": "website",
+                   "value": "http://app.stitchdata.com"
+                 },
+                 {
+                   "property": "company",
+                   "value": "Talend"
+                 },
+                 {
+                   "property": "phone",
+                   "value": "555-122-2323"
+                 },
+                 {
+                   "property": "address",
+                   "value": "25 First Street"
+                 },
+                 {
+                   "property": "city",
+                   "value": "Cambridge"
+                 },
+                 {
+                   "property": "state",
+                   "value": "MA"
+                 },
+                 {
+                   "property": "zip",
+                   "value": "02139"
+                 }
+               ]
+             }
+        
+        # generate a contacts record
+        response = self.post(url, data)
+        records = [response]
+        return records
+
+    def create_campaigns(self):
+        """
+        TODO couldn't find endpoint...
+        """
+        record_uuid = str(uuid.uuid4()).replace('-', '')
+
+        url = f"{BASE_URL}"
+        data = {}
+        
+        # generate a record
+        response = self.post(url, data)
+        records = [response]
+        return records
```

Comment:
Does this `TODO` mean it's not implemented?

If so, can we be consistent and `raise NotImplementedError()`?
If not, then can we remove the `TODO`?